src/server/components/lists/searches/translators-interpreters.ts

```diff
@@ -47,7 +47,9 @@ const serviceTypeToNoun: Record<string, string> = {
 
 function makeResultsTitle(country: string, servicesProvided: string[]): string {
   const sanitisedServicesProvidedQuery = servicesProvided.map((service) => serviceTypeToNoun[service]).filter(Boolean);
-  return `${sanitisedServicesProvidedQuery.join(" or ")} in ${country}`;
+  const formattedService =
+    sanitisedServicesProvidedQuery.length > 1 ? "Translators and interpreters" : sanitisedServicesProvidedQuery[0];
+  return `${formattedService} in ${country}`;
```

Comment:
when a user selects translators - but not interpreters, is the array (sanitisedServicesProvidedQuery) coming back with both service types? I'm not sure why a ternary is needed as well as the map. I think changing " or " to " and " would've done it

Reply:
Yep, there's probably a way to do this that I couldn't figure out at the time but basically if I just replaced 'or' with 'and' then the text would be 'Translators and Interpreters' which is incorrect because Interpreters has an uppercase I.

The ternary is purely for the lowercase I, but as I said I'm not sure if there's a better way to do that.